cluster/charts/rook-ceph/dashboards/ceph-cluster.json

```diff
@@ -0,0 +1,2602 @@
+{
```

Comment:
This is for a grafana dashboard, right? "dashboard" would be easily confused with the Ceph dashboard. How about naming this directory "grafana"?

Reply:
done